releasenotes/README.md

```diff
@@ -155,83 +35,94 @@ Run `sfdx plugins --core` to display the version of Salesforce CLI and the insta
+  * Command `force:package:version:list` displays this info (boolean, whether or not metadata was removed) when used with --verbose param
+  * Command `force:package:version:report` displays this info (boolean, whether or not metadata was removed).
+  * Command `force:package:version:promote` displays a warning if the version being promoted contains removed components and if --noprompt is not specified.
 
 ### Other Changes
 
-* When parsing the `.forceignore` file, Salesforce CLI uses the same rules and patterns as [git uses with the `.gitignore` file](https://git-scm.com/docs/gitignore). As a result, some entries in your existing `.forceignore` files might be invalid. The `force:source:*` commands warn you about any invalid entries.  
+* FIX: The `force:user:password:generate` no longer fails whe run against Spring '21 release of Salesforce.
 
-* FIX: The `force:source:convert` command works correctly when you individually convert two or more custom fields of the same custom object.  
+  The cause of the error, "The requested Resource does not exist API 51.0", is due the deprecation of a security setting, "enableSetPasswordInApi", a scratch org required in order to generate a password.
 
-    For example, let’s say your custom object Flower__c has two custom fields: Color__c and Size__c. Previously, this command would convert only one custom field, now it correctly converts all specified fields:
+  Starting in Spring '21, EnableSetPasswordInApi is a feature in your scratch org definition file and not a setting.
 
-    ```bash
-    $ sfdx force:source:convert -m CustomField:Flower__c.Color__c,CustomField:Flower__c.Size__c
-    ```
+* FIX: The `force:user:password:generate` no longer fails when generating a password for a user that does not have access to Profile Standard Object.
 
-    This fix also applies when using the `-x` parameter to specify a manifest file that lists these fields individually. The fix also applies to `force:source:deploy`, which executes a convert before it deploys.  ([GitHub issue #628](../../../issues/628))
-    
-* As [previously announced](./v49.md#49140-october-8-2020), we've reorganized the Salesforce DX Developer and Setup Guides to improve their usability and better align with how the Salesforce developer tools have evolved over time. In particular, we've moved most of the CLI Configuration and Tips topics to the [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm) and the Dev Hub topics to the [Developer Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_intro.htm). You might need to refresh your bookmarks.  
+* FIX: The output of `force:user:create` with `--json` now matches previous version.
```

Comment:
Edit:  "The output of `force:user:create --json` now matches the output from previous versions."

Reply:
you missed this edit from my last review :)